tests/cache.rs

```diff
@@ -61,8 +61,8 @@ fn it_should_return_the_file_if_client_has_old_version() {
     req.headers.set(IfModifiedSince(time::at(one_hour_ago)));
 
     match st.call(&mut req) {
```

Comment:
These matches could be replaced with `.map(do the check).unwrap()`, but your call at this point.